tools/ld-chroma-decoder/main.cpp

```diff
@@ -247,7 +277,7 @@ int main(int argc, char *argv[])
 
     // Positional argument to specify output video file
     parser.addPositionalArgument("output", QCoreApplication::translate("main", "Specify output file (omit or - for piped output)"));
-
+    
```

Comment:
Remove this whitespace?

Reply:
Fixed, along with at lines 350 and 491 in this file, and line 582 in lddecodemetadata.cpp.

Please note that the whitespacing in the source file is already inconsistent, as re-saving the relevant source files in an editor with "Remove trailing whitespace" enabled results in a considerable amount of diffs.

Reply:
I can only see one line with trailing whitespace in current master (in palencoder.cpp, so it's my fault).